index.html

```diff
@@ -356,6 +342,51 @@ <h2 class="lets-chat">Let's chat!</h2>
         </div>
       </div>
     </footer>
-    <script src="script.js"></script>
-  </body>
-</html>
+  </div>
+
+  <div class="modal" id="modal">
+    <div class="modal-header flex">
+      <div class="title">Multi - Post stories</div>
+      <button type="button" data-close-button class="close-button">&times;</button>
+    </div>
+    <div class="snapshoot"></div>
+    <div class="modal-body">
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus aspernatur tempore voluptatum id nihil ipsum culpa placeat quibusdam perferendis incidunt dicta neque, mollitia in quo provident eum rerum porro et est nulla. Esse molestiae fuga ad incidunt facere assumenda excepturi repellendus atque accusantium officia consectetur laborum qui eum aut, dolorum voluptatum doloribus asperiores quos recusandae a? Quo dignissimos magni suscipit similique aspernatur iusto nisi velit natus quod sed. Reiciendis mollitia quisquam hic architecto tempora vero ex cupiditate deserunt voluptatem corporis.
+    </div>
+
+    <div class="vertical-lines flex">
+      <div class="vertical-line"></div>
+      <p>html</p>
+      <div class="vertical-line"></div>
+      <p>Ruby on rails</p>
+      <div class="vertical-line"></div>
+      <p>css</p>
+      <div class="vertical-line"></div>
+    </div>
```

Comment:
- [ ] Please notice that this hard-coding HTML isn't fully functional in this case of the project because it's only one div (modal) for all the projects. Instead, we want to add several projects to the portfolio, each with its own data (image, paragraph, used techniques, and links), am I right? So, to accomplish this requirement, I highly suggest using the power of [Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array). This array is going to contain multiple objects (projects), and each object with its own data, which will be ready to be called once an action (clicking this project's "see more" button) occurs.

_This is a small piece of code as an example hint to start by._
```
const projects = [
  {
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
  },
  {
    // next object's (project) properties
  },
  {
    // next object's (project) properties
  },
]
```

- [ ] Also, please keep in mind this >> The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads)

_It means that you should create each project's data dynamically and append them in its own modal._

Reply:
Thanks for the feedback, I'm putting hands on it 🙏🏼